doc/flame/collision_detection.md

```diff
@@ -346,6 +346,10 @@ extra information like the distance, the normal and the reflection ray. The seco
 works similarly but sends out multiple rays uniformly around the origin, or within an angle
 centered at the origin.
 
+By default `raycast` and `raycastAll` scans for the nearest hit irrespective of how far it lies from
+the ray origin. But some use cases might be interested in finding hits only within a certain range.
```

Comment:
```suggestion
the ray origin. But some use cases, it might be interesting to find hits only within a certain range.
```

Reply:
Made it "But **in** some use cases, it might be interesting to find hits ..."